lib/binding_web/tsconfig.json

```diff
@@ -18,18 +18,20 @@
     "strictPropertyInitialization": true,
     "noImplicitThis": true,
     "alwaysStrict": true,
-    "noUnusedLocals": true,
     "noUnusedParameters": true,
     "noImplicitReturns": true,
     "moduleResolution": "node",
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "skipLibCheck": true,
+    "composite": true,
+    "isolatedModules": true,
   },
   "include": [
     "src/**/*",
     "script/**/*",
     "test/**/*",
+    "lib/**/*.ts"
```

Comment:
and explain this, is it needed? I don't mind, but I'd like to understand what the benefit is

Reply:
I added it so that `lib/web-tree-sitter.d.ts` it is type-checked also when running `tsc --build`